packages/create-sitecore-jss/src/templates/angular/src/app/jss-context.service.ts

```diff
@@ -16,14 +15,17 @@ export const jssKey = makeStateKey<JssState>('jss');
 export class JssContextService {
   // components can subscribe to this (or use getValue()) to get access to latest data from Layout Service,
   // as well as current language and server route
-  state: BehaviorSubject<JssState>;
-
-  constructor(protected transferState: TransferState, protected layoutService: JssLayoutService) {
-    this.state = new BehaviorSubject<JssState>(new JssState());
+  get state() {
```

Comment:
This is done to make upgrades easier. All the jss context usage is currently tied to JssContextService. Less hassle if instead of new injection/import we provide the option to reuse the same class, with slightly different getter.